x-pack/legacy/plugins/siem/public/components/super_date_picker/index.tsx

```diff
@@ -174,27 +184,28 @@ export const SuperDatePickerComponent = class extends Component<
       !isPaused &&
       (!this.state.isQuickSelection || (this.state.isQuickSelection && this.props.toStr !== 'now'))
     ) {
-      this.refetchQuery(this.props.refetch);
+      this.refetchQuery(this.props.queries);
     }
   };
 
-  private refetchQuery = (query: inputsModel.Refetch[]) => {
-    query.forEach((refetch: inputsModel.Refetch) => refetch());
-  };
-
-  private formatDate = (
-    date: string,
-    options?: {
-      roundUp?: boolean;
-    }
-  ) => {
-    const momentDate = dateMath.parse(date, options);
-    return momentDate != null && momentDate.isValid() ? momentDate.valueOf() : 0;
+  private refetchQuery = (queries: inputsModel.GlobalQuery[]) => {
+    queries
+      .filter(q => q.id !== 'kql' && q.refetch != null)
```

Comment:
I don't think you need to filter queries for `q.id !== 'kql'` as the argument is always `this.props.queries`, which in mapStateToProps already has this filter on it